Planes/PassengerPlane.js

```diff
@@ -1,19 +1,15 @@
-const Plane = require('./Plane');
+const {Plane} = require('./Plane');
 
 class PassengerPlane extends Plane {
 
-    constructor(model, maxSpeed, maxFlightDistance, maxLoadCapacity, passengersCapacity) {
-        super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
-        this._passengersCapacity = passengersCapacity;
+    constructor(obj) {
```

Comment:
Please give more meaningful name to a parameter that is passed to the `constructor`.

Reply:
fixed